lib/azure/armrest/armrest_service.rb

```diff
@@ -336,6 +346,24 @@ def tenants
         JSON.parse(resp.body)['value'].map{ |hash| Azure::Armrest::Tenant.new(hash) }
       end
 
+      # The name of the file or handle used to log requests.
+      #
+      def log
+        configuration.log
+      end
+
+      # Sets the log to +output+, which can be a file or a handle.
+      #
+      # Note that because logging is handled via a single class variable
+      # in the underlying RestClient library, you cannot really create
+      # different logs for different service classes. All requests will
+      # go to the same log.
+      #
+      def log=(output)
+        configuration.log = output
+        RestClient.log = output
```

Comment:
Since the `RestClient.log` is a class method, we can also directly add `log` and `log=` as class methods to `ArmrestService`. No need for the additional option in configuration.